docroot/profiles/custom/cgov_site/modules/custom/cgov_syndication/src/Controller/CgovSyndicationController.php

```diff
@@ -21,14 +21,13 @@ protected function getModuleName() {
   /**
    * Return an HTML rendering of a syndicated content item to HHS.
    *
-   * @param Drupal\node\Entity\Node $node
+   * @param mixed $node
```

Comment:
Why is this mixed?

Reply:
this is mixed because in this controller we are passing node object and this node object can be empty also if node id doesn't exist in the system.